src/Tribe/Aggregator/Cron.php

```diff
@@ -567,4 +569,22 @@ public function purge_expired_records() {
 			}
 		}
 	}
+
+	/**
+	 * Tries to fetch the data for the scheduled import and immediately process it.
+	 *
+	 * @since TBD
+	 *
+	 * @param $Tribe__Events__Aggregator__Record__Abstract $record
```

Comment:
Minor typo - `Tribe__Events__Aggregator__Record__Abstract` shouldn't be prefixed with a `$` sigil